extensions/identity-hub/src/test/java/org/eclipse/dataspaceconnector/identityhub/api/IdentityHubControllerTest.java

```diff
@@ -151,6 +152,8 @@ private RequestObject createRequestObject(String method, byte[] data) {
                                 .descriptor(Descriptor.Builder.newInstance()
                                         .method(method)
                                         .nonce(NONCE)
+                                        .recordId(UUID.randomUUID().toString())
+                                        .dateCreated(Instant.now().getEpochSecond())
```

Comment:
Should this be millis or is there a specific reason why this is the epoch date?

Reply:
From what I understand from the Decentralized Identity Foundation it should be epoch seconds: https://identity.foundation/decentralized-web-node/spec/0.0.1-predraft/#write

> The object MUST contain a dateCreated property, and its value MUST be an [Unix epoch timestamp](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap04.html#tag_04_16) that MUST be set and interpreted as the time the logical entry was created by the DID owner or another permitted party.

Reply:
OK thx